### DISCUSSION TITLE
Adding multiple remotes via the GUI

### DISCUSSION THREAD

**Discussion by LightningLion500** [community/community#124313] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi all!

Is it possible to add multiple remotes for 1 repository in the GUI like you can in Git via the terminal?
I would like to have 1 folder on my machine and 2 remote repositories (one on GitHub and one on Azure DevOps).

Can someone please explain to me how to do this?

**Comment by shivamgaur99** [comment]
Yes, it's possible to add multiple remotes to a repository in Git using the GUI interface. Here's how you can do it:

Open the Repository in Your GUI: This could be GitHub Desktop, Sourcetree, or any other Git GUI tool you prefer.

Navigate to Repository Settings or Options: Look for an option to manage remotes or repository settings. It's usually accessible from the toolbar or a menu.

Add a New Remote: Find the option to add a new remote repository. You'll typically need to provide a name for the remote (like "origin" for GitHub) and the URL of the repository.

Repeat for Each Remote: If your GUI tool allows it, repeat the process to add additional remotes for Azure DevOps or any other services you need.

Verify Remotes: Once added, you should see the list of remotes associated with your repository in the GUI. You can verify this by checking the repository settings or options again.

By following these steps, you can have one folder on your machine linked to multiple remote repositories, allowing you to push and pull changes to/from different platforms using the GUI interface.

**Reply by LightningLion500** [reply]
@shivamgaur99 Good morning!

Thank you for responding. This does not answer my question however.

I use GitHub Desktop and found no options to add multiple remotes. The only option is to edit the default "origin" remote (see image). Note that I sensored the actual URL.
![image](https://github.com/community/community/assets/70585970/83f0751c-df86-4e3b-83a4-7f7fef5dac64)

Also, your response seems AI generated and I, of course, already queried an LLM for this problem.

It seems like I can't create or see multiple remotes in the GitHub Desktop GUI. Could you please confirm this?

**Reply by shivamgaur99** [reply, marked as answer]
You're right. GitHub Desktop does not support

**Reply by shivamgaur99** [reply]
Although GitHub Desktop doesn't have a GUI option for managing multiple remotes, using the terminal within GitHub Desktop allows you to add and manage them effectively. Once the remotes are added via the terminal, GitHub Desktop will acknowledge and work with these remotes seamlessly.

**Reply by LightningLion500** [reply]
I can only see the other remotes in the "Branches" list after creating them in the terminal. 
![image](https://github.com/community/community/assets/70585970/90d95c79-ca22-4407-a980-f1e8c6ec20c4)
Being able to manage these remotes and pushing and fetching to all remotes would be a valuable addition to the GitHub Desktop UI, since this is not yet possible.

Anyway, I solved my problem by using the terminal. But I'd rather have this functionality in the GUI to have a more clear overview of changes, remotes and commit history.

**Reply by shivamgaur99** [reply]
Hopefully, the GitHub Desktop team will consider implementing it in future updates.
